src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrForwardFlower.cs

```diff
@@ -228,6 +236,7 @@ protected override async Task<bool> SameDirectionFlowAsync(
             return await _vmrUpdater.UpdateRepository(
                 mapping,
                 build,
+                patchExclusionFilters: PatchExclusions,
```

Comment:
I am not sure I like this new parameter:
1. The name could be better - something about excluding files? We're not excluding patches?
2. Do we even need this parameter? Don't we already have exclusions in the mapping? Can't we just add those there? `mapping = mapping with { Exclude = ... }`?

Reply:
I feel like if we use `mapping = mapping with { Exclude = ... }` it's a  bit "hidden". Maybe we just add it to source-mappings in general?

Reply:
I renamed it to `updateFileExclusionList` for now

Reply:
`additionalFileExclusions`?